### ISSUE TITLE
Use of 'builtin' masks check for cd error checking

### ISSUE THREAD

**Issue by matthewpersico** [koalaman/shellcheck#1234]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2164
- My shellcheck version (`shellcheck --version` or 'online'): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [ ] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/usr/bin/env bash

cd ~/foo

builtin cd ~/foo

```

#### Here's what shellcheck currently says:
```
Line 3	SC2164: Use 'cd ... || exit' or 'cd ... || return' in case cd fails.
```
#### Here's what I wanted or expected to see:

```
Line 3	SC2164: Use 'cd ... || exit' or 'cd ... || return' in case cd fails.
Line 4	SC2164: Use 'cd ... || exit' or 'cd ... || return' in case cd fails.
```

I don't believe that use of the builtin command should mitigate the check for cd failure.

**Comment by koalaman** [comment]
Fixed by @ngzhian. Thanks!
